float64 timeLimit
---
bool success
